pkg/bench/rttanalysis/orm_queries_bench_test.go

```diff
@@ -162,11 +162,56 @@ WHERE
 		},
 
 		{
-			Name:  "has_table_privilege real table",
+			Name:  "has_schema_privilege single",
+			Setup: `CREATE SCHEMA s`,
+			Stmt:  `SELECT has_schema_privilege('s', 'CREATE')`,
+		},
+
+		{
+			Name: "has_schema_privilege multiple",
```

Comment:
nit instead of multiple, state how many schemas are being created

we should probably test 1,3,5 schemas and see the relative increases

Reply:
Is there a way to create tables/schemas/etc using generate_series (or something similar) to avoid copying and pasting?

Reply:
I don't think we can use the sql language to iterate and do multiple DDLs but perhaps you could extend make a helper function to create a string for setup